cloudamqp/resource_cloudamqp_instance.go

```diff
@@ -217,7 +217,7 @@ func getPlanType(plan string) string {
 	switch plan {
 	case "lemur", "tiger":
 		return "shared"
-	case "bunny", "rabbit", "panda", "ape", "hippo", "lion":
+	case "squirrel", "bunny", "rabbit", "panda", "ape", "hippo", "lion", "rhino":
```

Comment:
The new Plan Names are 
`squirrel-1`, `bunny-1`, `bunny-3` etc

`nodes` is now deprecated according to the API docs https://docs.cloudamqp.com/#instances

`squirrel` is not an accepted plan name and will return a 400 error

`bunny` is the old plans which are going to be phased out. The new single node bunny plan is called `bunny-1` while a 3 node bunny plan is `bunny-3`

Example output from using `squirrel`

```
2020-12-01T16:04:49.527+0800 [DEBUG] plugin.terraform-provider-cloudamqp_v1.8.4: 2020/12/01 16:04:49 [DEBUG] go-api::instance::create params: map[name:instance-name no_default_alarms:false nodes:1 plan:squirrel region:amazon-web-services::us-east-1 rmq_version:3.8.9 tags:[dev]]
2020-12-01T16:04:51.114+0800 [DEBUG] plugin.terraform-provider-cloudamqp_v1.8.4: 2020/12/01 16:04:51 [DEBUG] go-api::instance::waitUntilReady data: map[]
2020/12/01 16:04:51 [DEBUG] cloudamqp_instance.NAME: apply errored, but we're indicating that via the Error pointer rather than returning it: CreateInstance failed, status: 400, message: map[error:Invalid value for parameter plan]
```

You can test this directly using the same API using `curl`

```sh
curl -u :$API_KEY \
  -d "name=test-instance&plan=bunny&region=amazon-web-services::us-east-1&tags=dev&rmq_version=3.8.9" \
  https://customer.cloudamqp.com/api/instances

curl -u :$API_KEY \
  -d "name=test-instance&plan=bunny-1&region=amazon-web-services::us-east-1&tags=dev&rmq_version=3.8.9" \
  https://customer.cloudamqp.com/api/instances
```